### ISSUE TITLE
Texture/Shadow Flickering on Banners, signs and foliage.

### ISSUE THREAD

**Issue by Mick458** [zesterer/openmw-shaders#24]
I get an odd flickering effect as soon as I install the Zesterer Shaders.
You can see the flickering on foliage, Banners, Signs and on shadows under the eaves of buildings.
This doesn't happen with Wazabears shaders or the default shaders.
I am running the latest dev build  (a42d09f366) and Zesterer shaders.
Attached video shows the issue,

https://user-images.githubusercontent.com/10111667/188288474-31ca1053-f4c2-4db2-acd5-77fc8af4f5b6.mp4

**Comment by XC3N** [comment]
I think you are not running the latest (0.49) build of openMW, can you confirm?

**Comment by XC3N** [comment]
just tested with latest 0.49, I cannot reproduce the issue. 

Again, this is something that happens when you run 0.47, can you confirm which you are running? latest 0.49 is 80d52c3da8, I cannot find a42d09f366

**Comment by Mick458** [comment]
> just tested with latest 0.49, I cannot reproduce the issue.
> 
> Again, this is something that happens when you run 0.47, can you confirm which you are running? latest 0.49 is 80d52c3da8, I cannot find a42d09f366

I just downloaded todays dev build and I am still getting the same issue.
For some reason the launcher isn't showing the Version number though
![image](https://user-images.githubusercontent.com/10111667/189619837-4f9604e0-9653-458f-909e-6f8e80b30c81.png)

**Comment by XC3N** [comment]
yeah I saw the same thing with the latest build, but the build number is shown on the main menu screen in-game.

Ok, what about mods? can you screenshot your openMW visual settings pages?

**Comment by Mick458** [comment]
This is the version I tested with
![image](https://user-images.githubusercontent.com/10111667/190059689-81f9134b-44f9-4d2b-869d-593c3a85f54a.png)

**Comment by Mick458** [comment]
![image](https://user-images.githubusercontent.com/10111667/190059788-06b08a25-16b6-457d-b6f7-2cc677e92b73.png)

**Comment by zesterer** [comment]
Is this still an issue?
